plugins/scaffolder-backend/README.md

```diff
@@ -64,6 +64,67 @@ you will not have any templates available to use. These need to be [added to the
 To get up and running and try out some templates quickly, you can or copy the
 catalog locations from the [create-app template](https://github.com/backstage/backstage/blob/master/packages/create-app/templates/default-app/app-config.yaml.hbs).
 
+## Configuration
+
+### Default Environment
```

Comment:
Can we make sure this gets added to the docs over here please? https://backstage.io/docs/features/software-templates/configuration